tonic/models/vic/grid_params.py

```diff
@@ -1648,3 +2150,369 @@ def lake(lake_file, soil_dict, max_numnod=10,
 
     return new_lake_dict
 # -------------------------------------------------------------------- #
+
+
+# -------------------------------------------------------------------- #
+def state(state_file, soil_dict, veg_dict, lake_dict, version_in='4.2',
+          nlayers=3, snow_bands=5, veg_classes=11, soil_nodes=3, nfrost=1,
+          max_numnod=10, cells=None, carbon=False):
+    """
+    Read the state file from state_file.  Also reorders data to
+    match gridcell order of soil file.
+    """
+
+    veg_classes_state = veg_classes + 1
+    if lake_dict is not False:
+        lakes = True
+    else:
+        lakes = False
+
+    print('reading {0}'.format(state_file))
+
+    with open(state_file) as f:
+        lines = f.readlines()
+
+    gridcel = np.zeros(cells).astype(int)
```

Comment:
No good reason for it - the reason is because I'm still new to python.  I'll fix it.